textproc/hyperscan/Portfile

```diff
@@ -37,6 +37,8 @@ github.tarball_from archive
 # https://github.com/intel/hyperscan/issues/197
 supported_archs     x86_64
 
+conflicts           vectorscan
+
 # Use ${python.default_version}
 # name consistency with ${prefix}/var/macports/sources/rsync.macports.org/macports/release/tarballs/ports/_resources/port1.0/group/python-1.0.tcl
 set python_default_version 311
```

Comment:
this should be using Python 3.12 if possible

Reply:
Thank you. Fixed.